src/glaciers/glacier/glacier2D_utils.jl

```diff
@@ -233,11 +235,12 @@ function init_gdirs_scratch(rgi_ids::Vector{String}, params::Parameters; velocit
 end
 
 # [Begin] Glathida Utilities
-function get_glathida!(gtd_file, gdirs, params; force=false)
+function get_glathida!(gtd_file, gdirs; force=false)
     glathida = pd.HDFStore(gtd_file)
     gtd_grids = map(gdir -> get_glathida_glacier(gdir, glathida, force), gdirs)
 
-    missing_glaciers = load(joinpath(params.simulation.working_dir, "data/missing_glaciers.jld2"))["missing_glaciers"]
+     # Update missing_glaciers list before removing them
+    missing_glaciers = load(joinpath(pwd(), "data/missing_glaciers.jld2"))["missing_glaciers"]
```

Comment:
This path should use one of the relative paths specified in Sleipnir, instead of `pwd()`.

Reply:
Hi @vivekag7, you should actually stick to the previous path that I was using. You need to use the `working_dir`, which is the path specified by the user in order to store files. If you give the root dir of Sleipnir it will save the files in the place where Julia stores the packages, which doesn't make sense.